x/oracle/alias_functions.go

```diff
@@ -0,0 +1,19 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// GetOracleAccount returns oracle ModuleAccount
+func (k Keeper) GetOracleAccount(ctx sdk.Context) authtypes.ModuleAccountI {
```

Comment:
I like this auxiliary method @toteki -- I think we should apply it to the x/leverage module as well when needing to get the module account.